packages/amplify-cli/src/domain/amplify-toolkit.ts

```diff
@@ -83,11 +82,11 @@ export class AmplifyToolkit {
     return this._buildResources;
   }
   get confirmPrompt(): any {
-    this._confirmPrompt = this._confirmPrompt || require(path.join(this._amplifyHelpersDirPath, 'confirm-prompt'));
+    this._confirmPrompt = this._confirmPrompt || require(path.join(this._amplifyHelpersDirPath, 'confirm-prompt')).confirmPrompt;
```

Comment:
You mean for the whole AmplifyToolkit? I'll measure the possible benefit that this dynamic import give for CLI command execution and if it is significant we leave it as-is, if there are no significant advantages it can be removed as a whole later.